### PR TITLE
update deps, docs, use standard

### PR DESCRIPTION
**Pull request by ungoldman**
## Changes

- update deps (readable-stream), dev deps (tap)
- use standard for linting
- simplify npm test script
- drop extraneous dev deps (jshint, nave)

## Version

major

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Todd Kennedy
+Copyright (c) 2018 Todd Kennedy
```
Comment (toddself) [inline LICENSE]: Add your name here!

**Comment by toddself** [comment]
Thanks! This is great! That tap version jump...

**Review by toddself — APPROVED** [review]

**Comment by toddself** [comment]
I can’t publish a new version from my phone so bug me tomorrow :)

**Comment by ungoldman** [comment]
No worries I want to run it through a couple things before I take off the
WIP label anyway. Thanks for taking a look!

**Comment by ungoldman** [comment]
Ok I think this is good to go. Looks like travis is inactive for this repo right now though:

https://travis-ci.org/toddself/image-validator-stream

**Pull request merged**